mlxtend/evaluate/tests/test_bootstrap_point632.py

```diff
@@ -113,11 +121,27 @@ def test_allowed_methods():
 
 
 def test_scoring():
-    from sklearn.metrics import f1_score
+    from sklearn.metrics import f1_score, roc_auc_score
     lr = LogisticRegression(solver='liblinear', multi_class='ovr')
     scores = bootstrap_point632_score(lr, X[:100], y[:100],
                                       scoring_func=f1_score,
                                       random_seed=123)
     f1 = np.mean(scores)
     assert len(scores == 200)
     assert np.round(f1, 2) == 1.0, f1
+
```

Comment:
Could you make a separate, new function for testing the `predict_proba=True` case (instead of adding it to the existing `test_scoring()` func)? It would make it easier to debug failures in future.